docs/develop/java/core-application.mdx

```diff
@@ -404,47 +404,79 @@ Ensure that your Workflow or Client can handle an Object type return or is able
 
 **How to customize your Activity Type using the Java SDK.**
 
-Activities have a Type that are referred to as the Activity name.
-The following examples demonstrate how to set a custom name for your Activity Type.
+Each Activity has a Type called the Activity 'name'.
+This name appears in the Workflow Execution Event History Summary, listed for each Activity Task.
+The name lets you identify Activity Types called during the Execution.
 
-The Activity Type defaults to method name, with the first letter of the method name capitalized, and can be customized using `namePrefix()` or `{ActivityMethod.name()}` to ensure they are distinct.
+Custom Activity Type names prevent name collisions across interfaces and Workflows.
+They offer descriptive Activity method names without concerns about re-using those names elsewhere in your project.
+They also support code management, especially in larger projects with many Activities.
+For example, you might use a prefix to group related activities together.
+Custom names also distinguish keys for gathering metrics without name conflicts.
 
-In the following example, the Activity Type defaults to `ComposeGreeting`.
+The following examples show how to set custom names for your Activity Type.
+
+**Default behavior**
+
+By default, an Activity Type is the method name with the first letter capitalized:
 
 ```java
 @ActivityInterface
 public interface GreetingActivities {
+    String sendMessage(String input);
+
     @ActivityMethod
```

Comment:
I recall getting feedback that we don't need to decorate the methods with `@ActivityMethod` _unless_ we are doing something like changing the name. It doesn't hurt anything, and personally I like having it but I've been convinced it's not necessary. All methods within an Activity interface are considered Activities.

Reply:
Ah, as I read further I see the point you were making. Lovely. Ignore the above.

Reply:
Should I mention that `sendMessage` explicitly inherits prefixes and is an `@ActivityMethod`? I think that might be a good thing.

Reply:
It's mentioned a lot already, so I added examples for that as well, to make it clear and punch it home.